Python/MagicPen/README.md

```diff
@@ -0,0 +1,5 @@
+Ever dreamed of just swing your pen 🖋️ in the air and it actually draws on the screen...Well that is exactly Magic Pen🔮 does using OpenCV and NumPy✨ .
+
+## Demo :
+
+![Demonstration](https://github.com/Prathyusha-Guduru/Data/blob/master/Magic%20pen.gif)
```

Comment:
You can add the GIF File in the Directory itself and give it a local reference instead of referencing it from an external source.

Reply:
> Keep your Commit Name meaningful as well. Keeping it as something like "Added Files by Upload" shows that you did not use the Git but the Github Web UI which ideally should not be used.

Got it...Will learn about git and use it for committing files and make the recommended changes soon! A little busy with mid-semester exams 😅